ads/_config.js

```diff
@@ -675,6 +675,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'onnetwork': {},
```

Comment:
It is highly recommended to implement the `renderStart` [API](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) to provide better user experience. Please let us know if there's any question. Thanks